cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -11,12 +11,12 @@
 # inability to use the software.
 #
 # FindRTIConnextDDS
-# -----------------
+# =================
 #
 # Find RTI Connext DDS libraries.
 #
 # Components
-# ^^^^^^^^^^
+# ----------
 # This module sets variables for the following components that are part of RTI
 # Connext DDS:
```

Comment:
Just say RTI Connext (no DDS)

(don't bother searching and replacing all ... we should update this as we can, and be consistent within a file ... if changing all instances in a file is too big a job, then leave it alone)